packages/prisma/postgresql/schema.prisma

```diff
@@ -122,6 +124,17 @@ model WorkspaceInvitation {
   workspace   Workspace     @relation(fields: [workspaceId], references: [id], onDelete: Cascade)
 }
 
+model WorkspaceAiFeature {
+  id           String       @id @default(cuid())
+  createdAt    DateTime     @default(now())
+  updatedAt    DateTime     @default(now()) @updatedAt
+  workspace    Workspace    @relation(fields: [workspaceId], references: [id], onDelete: Cascade)
+  workspaceId  String
+  prompt       String
+  credential   Credentials? @relation(fields: [credentialId], references: [id], onDelete: SetNull)
+  credentialId String?
+}
+
```

Comment:
Is it really necessary? I would probably just inline that in the Workspace model. Typebot has not a relation-heavy db schema.

Reply:
Yes, let's keep it simple and have one field for group title auto rename. Let's build for it without thinking of next AI features this should just implement #1577

In Typebot, defaults are always constants in the code.

Reply:
Okay, got it 💯 Trying to keep it simple and focusing on implementing #1577 for now. I will share my progress and revert back here if any further help is needed from you. Thank you @baptisteArno